WORKSPACE

```diff
@@ -18,7 +18,8 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_file")
 git_repository(
     name = "build_bazel_rules_apple",
     remote = "https://github.com/bazelbuild/rules_apple.git",
-    tag = "0.9.0",
+    commit = "19f031f09185e0fcd722c22e596d09bd6fff7944",  # 0.19.0
+    shallow_since = "1570721035 -0700",
```

Comment:
Tip: Nit: Please document the date/time for this number.

Reply:
Done.